src/main/java/jaci/openrio/toast/lib/registry/Registrar.java

```diff
@@ -32,7 +32,7 @@
      */
     public synchronized  <T extends Type> T fetch(ID id, Class<T> clazz, Supplier<T> creator) {
         Type in_register = registered.get(id);
-        if (in_register == null) {
+        if (in_register.equals(null)) {
```

Comment:
null.equals(null) will cause a nullpointer exception. Use the original `in_register == null` instead